release.sh

```diff
@@ -1,6 +1,6 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
I'm fine with the `$@` change but can you explain this one more? When will we have a setup that maintainers don't have `bash` yet have `sh`?